.github/actions/cargo-install-upload-artifacts/action.yml

```diff
@@ -13,10 +13,8 @@ runs:
       run: |
```

Comment:
Also, the title on this action (on L11) should probably be changed, since we no longer check if the crate has binaries.